sources/draft-camelot-holy-grenade.adoc

```diff
@@ -561,24 +372,10 @@ tag::figure2[]
                      `''-----''`
 ....
 
-////
-// end::figure2a[]
-
-[.comment]
-end::figure2[]
-////
-
-
 [[sovereign-orb]]
 .The Sovereign's Orb made invisible
 [link=https://upload.wikimedia.org/wikipedia/commons/b/bf/Coa_Illustration_Elements_Globus_cruciger.svg,align=right]
-image::cruciger.svg[Orb,124,135]
-
-////
-[.comment]
-tag::index1[]
-// tag::index[]
-////
+image::cruciger.svg[]
```

Comment:
The attributes on image::[] are legal (alt, height, width); do they cause a problem?

Reply:
It seems this does not cause a problem, at least I am not reproducing it now.